.circleci/config.yml

```diff
@@ -15,18 +15,12 @@ parameters:
   node-orb-version:
     type: string
     default: "5.0.1"
-  python-cache-version:
-    type: string
-    default: v3
-  dev-cache-version:
-    type: string
-    default: v3
-  node-cache-version:
+  cache-version:
     type: string
     default: v3
```

Comment:
I still don't understand this default value. Is there an option to simply cache where the key is the computed hash of the files we care about when they change?

Reply:
It's just which algorithm to use (v3).

Reply:
So if I understand correctly, then I don't think we need to make this a pipeline parameter. Can be hardcoded. If we want to update the caching version then it should be a PR.